.storybook/utils/styled-system-props.js

```diff
@@ -140,20 +140,295 @@ const generateStyledSystemProps = (
   ];
 };
 
+const generateStyledSystemColorProps = (
+  defaults
+) => {
+  return [
+    {
+      name: 'color',
+      type: { summary: 'string' },
+      description: 'Color, theme value or any valid CSS string.',
+      required: false,
+      defaultValue: {
+        summary: defaults.color || '-'
+      }
+    },
+    {
+      name: 'bg',
+      type: { summary: 'string' },
+      description: 'Background, theme value or any valid CSS string.',
+      required: false,
+      defaultValue: {
+        summary: defaults.bg || '-'
+      }
+    },
+    {
+      name: 'opacity',
+      type: { summary: 'decimal | percentage' },
+      description: 'Any valid CSS string',
+      required: false,
+      defaultValue: {
+        summary: defaults.opacity || '-'
+      }
+    }
+  ];
+};
+
+const generateStyledSystemLayoutProps = (
+  defaults
+) => {
+  return [
+    {
+      name: 'width',
+      type: { summary: 'number | string' },
+      description: 'An integer multiplier of the base spacing constant (8px) or any valid CSS string.',
+      required: false,
+      defaultValue: {
+        summary: defaults.width || '-'
+      }
+    },
+    {
+      name: 'height',
+      type: { summary: 'string' },
```

Comment:
Changed this to accurately reflect what styled-system allows

Reply:
It still only accepts `string` while according to https://styled-system.com/api#layout it accepts both strings and numbers (and arrays too, but I'm not sure if we should mention it because we don't have any breakpoints defined in theme)